uber/config.py

```diff
@@ -13,15 +13,9 @@ def mixin(cls, klass):
     def include_plugin_config(self, plugin_config):
         """Plugins call this method to merge their own config into the global c object."""
 
-        for attr, val in plugin_config.items():
-            if not isinstance(val, dict):
-                setattr(self, attr.upper(), val)
-
-        if 'enums' in plugin_config:
-            self.make_enums(plugin_config['enums'])
-
-        if 'dates' in plugin_config:
-            self.make_dates(plugin_config['dates'])
```

Comment:
On Wed, Oct 14, 2015 at 11:08 PM, Eli Courtwright <notifications@github.com>
wrote:

> Thaeli mentioned earlier taking the c object definition out of runtime
> into a separate command, something like sep load_config_options.
>
> That seems like a huge refactor, much more work than just modifying
> existing options in-place in the c.make_enum() method. I'm theoretically
> open to anything since the proof is in the pudding as far as
> clean/intuitive/working code, but I wouldn't want to do anything that
> drastic just to support enum merging.
>

Well, I need it for configDB. The current setup has parsing the config
files at runtime very intertwined with the implementation of the config
dict.